src/chordsymbol.ts

```diff
@@ -628,33 +351,12 @@ export class ChordSymbol extends Modifier {
   }
 
   /** Add a line of the given width, used as a continuation of the previous symbol in analysis, or lyrics, etc. */
-  // eslint-disable-next-line
-  addLine(width: number, params: any = {}): this {
-    const symbolType = SymbolTypes.LINE;
-    return this.addSymbolBlock({ ...params, symbolType, width });
-  }
-
-  /**
-   * Set the chord symbol's font family, size, weight, style (e.g., `Arial`, `10pt`, `bold`, `italic`).
-   *
-   * @param f is 1) a `FontInfo` object or
-   *             2) a string formatted as CSS font shorthand (e.g., 'bold 10pt Arial') or
-   *             3) a string representing the font family (one of `size`, `weight`, or `style` must also be provided).
-   * @param size a string specifying the font size and unit (e.g., '16pt'), or a number (the unit is assumed to be 'pt').
-   * @param weight is a string (e.g., 'bold', 'normal') or a number (100, 200, ... 900).
-   * @param style is a string (e.g., 'italic', 'normal').
-   *
-   * @override See: Element.
-   */
-  setFont(f?: string | FontInfo, size?: string | number, weight?: string | number, style?: string): this {
-    super.setFont(f, size, weight, style);
-    this.textFormatter = TextFormatter.create(this.textFont);
-    return this;
-  }
-
-  setEnableKerning(val: boolean): this {
-    this.useKerning = val;
-    return this;
+  addLine(
+    params: Partial<{
+      symbolModifier: SymbolModifiers;
+    }> = {}
+  ): this {
+    return this.addText(String.fromCharCode(0xe874, 0xe874), params);
```

Comment:
I'm beginning to think that we need comments for each SMuFL unicode codepoint that we stick in our code base. 

`0xe874` means nothing to me :-)

Maybe we should always add a comment above lines that include SMuFL code points?

```ts
// SMuFL U+E874 / csymMinor / Minor
return this.addText(String.fromCharCode(0xe874, 0xe874), params);
```

If you do it for this one, I can help out with other ones by submitting PRs. I think having this comment will help future readers of our code.

Reply:
Changed like other comments in the file with:
/*csymMinor*/